terraform/rds-databases/instance.tf

```diff
@@ -61,8 +61,8 @@ resource "aws_db_instance" "shared" {
   backup_retention_period      = 3
   storage_type                 = "gp3"
   engine                       = "postgres"
-  engine_version               = "16.3"
-  instance_class               = "db.t4g.micro"
+  engine_version               = "16.8"
+  instance_class               = "db.m7g.large"
```

Comment:
I think that @Mark-Simulacrum bumped the instance some time ago. Not sure about the Postgres version upgrade.

Reply:
probably the db is configured in a way that aws automatically updates to the latest version of postgres (minor version updates only)